doc/news/9.5.0-vs-9.5.2.h

```diff
@@ -0,0 +1,48 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 - 2023 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+/**
+@page changes_between_9_5_0_and_9_5_2 Changes between Version 9.5.0 and 9.5.2
+
+<p>
+This is the list of changes made between the release of deal.II version
+9.5.0 and that of 9.5.2. All entries are signed with the names of the
```

Comment:
Is it 9.5.1 or really 9.5.0? I missed that in the corresponding file on the branch (not that it is terribly important).

Reply:
It's really 9.5.0 because we (== I) forgot to create one for 9.5.1.